src/consensus/pbft/libbyz/Reply_stable.cpp

```diff
@@ -61,10 +61,7 @@ bool Reply_stable::verify()
   std::shared_ptr<Principal> p = node->get_principal(id());
   if (p)
```

Comment:
Same here, I think we can just return `true`?

Reply:
this is checking if we know of the principal